### PR TITLE
15133 create alter field migrations

### PR DESCRIPTION
**Pull request by santilin**
| Q             | A
| ------------- | ---
| New feature?  | yes
| Breaks BC?    | no
| Tests pass?   | untested
| Fixed issues  | #15133

### CODE REVIEW

File: docs/guide/db-migrations.md [hunk 1]
```diff
@@ -190,7 +190,7 @@ A list of all available methods for defining the column types is available in th
 
 ## Generating Migrations <span id="generating-migrations"></span>
 
-Since version 2.0.7 migration console provides a convenient way to create migrations.
+ version 2.0.7 migration console provides a convenient way to create migrations.
```
Comment (samdark) [inline docs/guide/db-migrations.md]: Should not be changed.

File: docs/guide/db-migrations.md [hunk 2]
```diff
@@ -497,7 +497,7 @@ yii migrate/create add_xxx_column_yyy_column_to_zzz_table --fields="xxx:integer,
 If the migration name is of the form `drop_xxx_column_from_yyy_table` then
 the file content would contain `addColumn` and `dropColumn` statements necessary.
 
-```php
+```
```
Comment (samdark) [inline docs/guide/db-migrations.md]: Should not be changed.

**Comment by samdark** [comment]
Please resolve conflicts so it will run unit tests against current code...

**Comment by developeruz** [comment]
Replaced with #16465 because this one is from `unknown repository` and probably can't be merged

**Pull request closed without merging**